packages/next/src/generators/init/generator.spec.ts

```diff
@@ -1,12 +1,26 @@
-import { addStacksAttributes } from '@ensono-stacks/test';
-import { Tree, readJson } from '@nx/devkit';
+import { tsMorphTree } from '@ensono-stacks/core';
+import {
+    addStacksAttributes,
+    checkFilesExistInTree,
+} from '@ensono-stacks/test';
+import { Tree, joinPathFragments, readJson } from '@nx/devkit';
 import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing';
 import { applicationGenerator } from '@nx/next';
 import { Schema as NextSchema } from '@nx/next/src/generators/application/schema';
 
 import generator from './generator';
 import { NextGeneratorSchema } from './schema';
 
+function snapshotFiles(tree, files: string[]) {
+    expect(() => checkFilesExistInTree(tree, ...files)).not.toThrowError();
+    const project = tsMorphTree(tree);
+    files.forEach(file => {
+        expect(project.addSourceFileAtPath(file).getText()).toMatchSnapshot(
+            file,
+        );
+    });
+}
+
 describe('next install generator', () => {
```

Comment:
I don't think there is any unit test coverage for checking package installation? Worth adding this:
e.g: 
```
it('should install deps into package.json', () => {
        const packageJson = readJson(appTree, 'package.json');
        expect(packageJson?.devDependencies).toMatchObject({
            cypress: CYPRESS_VERSION,
            '@cypress/grep': CYPRESSGREP_VERSION,
        });
    });
```

Reply:
added unit test